THIRD-PARTY-LICENSES

```diff
@@ -21,4 +21,228 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
\ No newline at end of file
+SOFTWARE.
+
```

Comment:
I've included the plugin it in the recent changes and added tests about the generated content to get aware of licence changes.

I decided to merge the d3-* related licences manually because I could not find a simple approach to fully automated that merge automatically in a reasonable amount of time without adding much extra build complexity. I suggest to start that automation when we are in a situation where we have to ship more 3rd party dependencies.